server/inc/API.php

```diff
@@ -28,6 +28,12 @@ public function __construct(DB $db)
 			$url .= ':' . $_SERVER['SERVER_PORT'];
 		}
 
+		// Check if environment variable with service url is set
+		// When running inside Docker, we cannot determine the correct URL by ourselves.
+		if (getenv('gpodder_url') !== false) {
+		    $url = getenv('gpodder_url');
```

Comment:
Should not be an environment variable but a constant defined in config.local.php

Reply:
Then you should create a config.local.php file specific to your docker distribution.

Reply:
I created a new BASE_URL constant, also available via local environment, please use this.